DanielVargas/tic_tac_toe/src/com/company/Tictactoe.java

```diff
@@ -0,0 +1,176 @@
+package com.company;
+import java.util.Scanner;
+
+public class Tictactoe {
+    final int cols=3;
+    final int rows=3;
+    private String [][] board;
+    private String player1;
+    private String player2;
+    private int num_of_plays;
+    Tictactoe(String player1, String player2) {
+        this.board = new String[this.cols][this.rows];
+        this.player1 = player1;
+        this.player2 = player2;
+        this.num_of_plays=0;
+        for (int i = 0; i < this.rows; i++) {
+            for (int j = 0; j < this.cols; j++) {
+                this.board[i][j] = "";
+            }
+        }
+    }
+    public void printboard()
```

Comment:
Please follow java convention to declare method names, when the name is compound for many words the first letter of the word is in lowercase the remaining ones are written with uppercase:

```java
public void printBoard() {
}
```